runtime/kusama/src/weights/pallet_utility.rs

```diff
@@ -29,6 +29,9 @@ impl<T: frame_system::Trait> pallet_utility::WeightInfo for WeightInfo<T> {
 		(18_450_000 as Weight)
 			.saturating_add((1_730_000 as Weight).saturating_mul(c as Weight))
 	}
+	fn batch_all(c: u32) -> Weight {
```

Comment:
I also have no idea what these `batch_all` impls are about; we should sort out what they mean and ensure that this is a reasonable impl before merging.